python/pacemaker/_cts/CTS.py

```diff
@@ -178,8 +202,24 @@ def wait_for_all_nodes(self, nodes, timeout=300):
 
 
 class Process:
+    """ A class for managing a process """
+
     def __init__(self, cm, name, dc_only=False, pats=None, dc_pats=None,
                  badnews_ignore=None, triggers_reboot=False):
+        """ Create a new Process instance.
+
+            Arguments:
+
+            cm              -- A ClusterManager instance
+            name            -- The command being run
+            dc_only         -- Should this process only run on the DC?
```

Comment:
"Should this daemon be killed only on the DC?"

Reply:
Fixed